src/snowcli/plugins/nativeapp/version/commands.py

```diff
@@ -41,19 +41,19 @@ def create(
     version: Optional[str] = typer.Argument(
         None,
         help=f"""The identifier or 'version string' of the version you would like to create a version and/or patch for.
-        Defaults to undefined, which means the CLI will use the version, if present, in the manifest.yml.""",
+        Defaults to undefined, which means the CLI uses the version, if present, in the `manifest.yml`.""",
```

Comment:
replace description with:

Version of the app package that you would like to create a version of patch. Defaults to the version specified in the `manifest.yml` file.

Reply:
Could you clarify: `Version of the app package that you would like to create a version of patch.`

Reply:
Should be version OR patch